<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Volume_widget</class>
 <widget class="QWidget" name="Volume_widget">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>80</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="Volume_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>270</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>270</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>270</width>
     <height>80</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="Volume_iconLabel">
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>65</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
      </property>
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Volume_LevelLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>65</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>65</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
